packages/www/pages/docs/guides/debugging-guide.mdx

```diff
@@ -0,0 +1,71 @@
+import DocsLayout from "../../../components/DocsLayout";
+
+<DocsLayout>
+
+
+# Debugging Livepeer.com Stream Playback
+
+This step-by-step guide will help you debug common stream playback problems. Try these steps before reaching out to Livepeer.com directly.
+
+We will use the [Livepeer.com test player](/app/test-player) to debug. You must be logged into the Livepeer.com dashboard. 
+
+#### Prerequisite: Locate your playback URL. 
+
+Your playback URL is specific to the stream ingest region. 
+- This [Livepeer.com API guide](/docs/guides/api/playback-a-live-stream) touches on finding your playback URL by combining a base url with a `playbackId`. If you don't know your stream `playbackId`, you can retrieve it via [`GET /stream/<id>`](/docs/api/stream/get-stream).
+- You can also find the playback URL on the Livepeer.com dashboard stream page, `https://livepeer.com/app/stream/<stream-id>`. Learn more [here](/docs/guides/dashboard/playback-a-stream).
+
+## Step 1
+
+Load your playback URL in the [Livepeer.com test player](/app/test-player). If you get a "Stream not found" error, confirm that your URl is correct. It should look something like this:  https://region-cdn.livepeer.com/hls/123456abcdef7890/index.m3u8
+
+## Step 2
+
+Does anything playback in *your* own player?
```

Comment:
Perhaps adding something like "Does anything playback in *your* player of choice? If you don't have a player you have used in the past, feel free to use [TheoPlayer](https://demo.theoplayer.com/test-your-stream-with-statistics). For instructions on how to use Theoplayer, see step 6 [here](https://livepeer.com/blog/first-livepeer-stream-in-five-minutes)." I'd be worried about ppl not knowing what "your player" means.

Reply:
I vaguely remember talking about there being a real benefit to testing on another player other than the one we selected. @ericxtang I think you were the one who talked about that? Could you give more details?

Reply:
Some other players offer logs or more information on latency. However, let's start with directing users to debug using one player, the livepeer.com player. it will be confusing for them to have to bop around among Livepeer, theo and/or bitmovin.

In future iterations of the livepeer.com test player, we will have more metrics about the network and ingest stream.